Explorer/Assets/DCL/Multiplayer/Connections/Messaging/Hubs/IMutableMessagePipesHub.cs

```diff
@@ -0,0 +1,11 @@
+using DCL.Multiplayer.Connections.Messaging.Pipe;
+
+namespace DCL.Multiplayer.Connections.Messaging.Hubs
```

Comment:
I think you can disable "provide namespace" for subfolders of `Messaging `folder and `Atchipelago `folder